pkg/rslog/README.md

```diff
@@ -12,3 +13,86 @@ Provides a logger that wraps `github.com/sirupsen/logrus`.
 - [testlog](../../examples/cmd/testlog) and
   [testservice](../../examples/cmd/testservice) also
   demonstrate using `rslog` for logging and debug logging.
+
+### Using and changing the default logger
+
+`rslog.NewDefaultLogger()` returns a singleton instance of a logger that can be used across multiple packages in your
+application. There are setter methods on the logger that can be used to change the default logger's behavior. These
+examples assume that you have `logger := rslog.NewDefaultLogger()`.
+
+```go
+// Set the default logger to use the JSON formatter
+logger.SetFormatter(rslog.JSONFormat)
+// Set the output to a file instead of stdout
+logger.SetOutput(os.OpenFile("mylogfile.log", os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0666))
+```
+
+Remember that this is a singleton instance. Changes you make in one file will affect the logger in other files.
+
+### Rotating log files
+
+Rotating log files is not specified as part of the Posit logging standards to be part of an application.
+It can be left to the host for utilities such as logrotate. However, should you prefer to implement it in
+your application, the way to do it is to implement a file writer that does the rotation. One way to achieve
+this is with the [lumberjack library](https://github.com/natefinch/lumberjack).
+
+```go
+logger := rslog.NewDefaultLogger()
+logger.SetOutput(&lumberjack.Logger{
+	Filename:   "/var/log/rstudio/myapp/server.log",
```

Comment:
Here, and below, consider using a more generic path that does not include `rstudio`.